test/e2e/basic_test.go

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) 2022 Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+
+package e2e
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+const (
+	adminIAMPolicyYaml string = "../resources/iam-policy-default.yaml"
+	adminIAMPolicyName string = "rule-of-two"
+	adminCRBYaml       string = "../resources/admin-crb.yaml"
+	otherIAMPolicyYaml string = "../resources/iam-policy-other.yaml"
+	otherIAMPolicyName string = "there-can-be-only-one"
+	otherCRBYaml       string = "../resources/other-crb.yaml"
+	otherRoleYaml      string = "../resources/other-clusterrole.yaml"
+)
```

Comment:
Would you please make these block scoped variables? I'm not sure there's a need to make them package-level.